### ISSUE TITLE
Unmapping hotkeys doesn't work

### ISSUE THREAD

**Issue by thth** [bluaxees/Cloe#27]
e.g. from default settings:
- changing `Start Capture` from `Q` to `<Unmapped>` and saving: the hotkey reverts back to `Q`
- changing `Open Settings` from `<Unmapped>` to `A`, saving, then changing `A` to `<Unmapped>` and saving: Cloe quietly crashes. Opening Cloe again shows `Open Settings` still set to `A`; attempting to change to `<Unmapped>` and saving again doesn't crash, but reverts the hotkey back to `A`.

`Restore Defaults` does work as expected though, so this isn't a huge deal 🙂

**Comment by ONEsuzuki** [comment]
Yep same problem here, reverted back to original key binds and it worked.
Just a note to the developers though, fantastic application. No more Manga_OCR command line, really simple well done.
